product-service-description/DE-business-model.md

```diff
@@ -0,0 +1,37 @@
+# Business Modell von fiskaltrust in Deutschland
```

Comment:
Vorschlag: "Das fiskaltrust Geschäftsmodell in Deutschland"

Reply:
ok